scripts/build_utils.sh

```diff
@@ -837,6 +837,16 @@ gen_debian_version() {
     echo "${raw}${bptag}"
 }
 
+setup_cmakeoptions_from_ci() {
+  if [-z BRANCH ];then
```

Comment:
$BRANCH will always be set.  Kind of unnecessary.  Even if it wasn't, returning 0 silently isn't ideal if the function is always called.  An `else echo "\$BRANCH is not defined"` would suffice.

Reply:
if that is the case, i am okay removing it